### PR TITLE
SONARJAVA-4841 Add S6898:  Avoid high frame rates on mobile

### PR DESCRIPTION
**Pull request by johann-beleites-sonarsource**

### CODE REVIEW

File: sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json [hunk 1]
```diff
@@ -499,6 +499,7 @@
     "S6857",
     "S6862",
     "S6863",
+    "S6898",
```
Comment (ADarko22) [inline sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json:502]: should remove it from sonar way now or on a following effort?

**Review by ADarko22 — APPROVED** [review]
Look good! Nice the corner case that tests that non-final non-static fields can lead to FN!

Small comment about removing it from Sonar Way
